src/styles/components/buttons.scss

```diff
@@ -26,7 +26,16 @@
     padding: 2.5rem;
 }
 
-@each $name, $type in $os-colors {
+.cta > .btn {
+    background-color: color(orange);
+    border: 0;
+    color: color(neutral-lightest);
+    font-variant: small-caps;
+    margin: 0 auto;
+    padding: 1.5rem 2rem;
+}
+
+@each $name, $type in $color-types {
```

Comment:
Hmm, this is what broke a lot of colors.  It looks like it was accidentally changed from `$os-colors` to `$color-types`.